content/deployment/reference/server-specifications.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Server specifications
+weight: 1
+---
+
+# Server specifications
+
+The minimum recommended server specifications are:
+
+- 2GB RAM
+- 1CPU @ 2GHz
+- 10GB storage
+
+Approximately 750MB of storage is required for the engine itself, with the remaining space allocated for data generated by the engine.
+
+## Storage growth
+
+Storage requirements typically grow at a rate of less than 0.5MB per tracked terms per month on average. This growth rate varies depending on the type of service being tracked:
+
+- Large digital services, like those in the [PGA collection](https://github.com/OpenTermsArchive/pga-declarations): ~1MB per terms per month
+- Standard collections: ~0.1-0.3MB per terms per month
+
+## Operating system
+
+Supported operating systems:
+- Ubuntu 20.04 or higher
+- Debian 11 or higher
+
+## Location
+
+To align with our [third design principle]({{< relref "concepts/design-principles#3-obtain-documents-like-a-user-would" >}}), servers should be deployed in geographic regions that align with the jurisdiction the collection targets. This is crucial because some services deliver different versions of documents based on the server's location, even when the documents are supposed to be identical. Therefore, ensure your server is physically located in a country within the jurisdiction that the collection targets.
```

Comment:
- Let's speak more about the user, less about us.
- Let's keep the neutral phrasings, not using “you”.

```suggestion
Servers should be physically located in geographic regions that align with the jurisdiction the collection targets to ensure to [obtain documents like a user would]({{< relref "concepts/design-principles#3-obtain-documents-like-a-user-would" >}}). This is crucial because some services deliver different versions of documents based on the location of the request, even when the documents are supposed to be identical.
```